vsphere/tests/test_check.py

```diff
@@ -32,6 +32,7 @@ def test_realtime_metrics(aggregator, dd_run_check, realtime_instance):
                 metric['name'], metric.get('value'), hostname=metric.get('hostname'), tags=metric.get('tags')
             )
 
+    aggregator.assert_metric('datadog.vsphere.collect_events.time', metric_type=aggregator.GAUGE, count=1)
```

Comment:
note: the metric was not sent before, because it was failing when trying to call `get_new_events`